koku/reporting/rate/models.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Models for identity and access management."""
+
+from uuid import uuid4
+
+from django.db import models
+
+
+class Rate(models.Model):
```

Comment:
This model could benefit from a unique constraint so that a rate can't be repeated in the db.

Reply:
That's tricky, because multiple rates can be applied to the same metric. 

So, I could do a unique_together on (metric, timeunit, price), but it doesn't really buy us much.